City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1524: add translations to attachments

# [AU-1524](https://helsinkisolutionoffice.atlassian.net/browse/AU-1524)


## What was done


* add translations to attachments

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-1524-conf-changes`
  * `make fresh`
* Run `make drush-cr`

## How to test
* Check that translations are correct


[AU-1524]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1524?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
